src/main/java/http/routes/MethodOptions.java

```diff
@@ -1,9 +1,6 @@
 package http.routes;
 
-import http.HTTPStatusCodes;
-import http.Response;
-import http.ResponseBuilder;
-import http.Route;
+import http.*;
```

Comment:
nitpick: Try to only import the files you need, rather than use `*` to import all of them/

Reply:
Refactored to use single class imports instead of `*`